apps/src/templates/manageStudents/utils.ts

```diff
@@ -0,0 +1,20 @@
+export function getFullName(student: {
+  name: string;
+  familyName: string;
+}): string;
+export function getFullName(name: string, familyName?: string): string;
+
+export function getFullName(
```

Comment:
This looks great. I was wondering 1) why we need the function type definitions above, since it seems like the type definition for this function is accurate and 2) why we need the if statement around the argument type and whether we can just split the if statement into two functions with the same name but different arg types.

Reply:
1) Initially I didn't have those either but was getting an error.  Some research lead me to this [documentation](https://www.typescriptlang.org/docs/handbook/2/functions.html#function-overloads).  My understanding of this is that because we have a different number of parameters, we need to include the function signatures separately.

2) This suggestion is how I have done it in Java too (and what I tried here but got errors). Using the documentation above, this is how I got to this implementation.